src/components/admin-sidebar/SideBar.js

```diff
@@ -38,14 +38,21 @@ const Sidebar = () => {
           <Link to="#">
             <IoIcons.IoIosLock /> <span> Permissions </span>
           </Link>
-          <Link to="#">
+
+          <NavLink to="#">
             <IoIcons.IoMdAnalytics /> <span> Actions </span>
-          </Link>
+          </NavLink>
         </div>
-        <Link to="#">
+        <NavLink to="/crud-route">
           <AiIcons.AiOutlineReload />
           <span> Routes </span>
-        </Link>
+        </NavLink>
+
+        <NavLink to="/crud-route">
+          <AiIcons.AiOutlineReload />
+          <span> Routes </span>
+        </NavLink>
+
```

Comment:
Here there is a deplications of Navlink "Routes" Please remove one .

Reply:
Yes I will but Make it so that other team member will replace incase they need it